lib/livebook_web/helpers.ex

```diff
@@ -375,4 +375,20 @@ defmodule LivebookWeb.Helpers do
 
   def file_system_label(%FileSystem.Local{}), do: "Local disk"
   def file_system_label(%FileSystem.S3{} = fs), do: fs.bucket_url
+  
+  def notebook_import_url(url) do
```

Comment:
```suggestion
  @doc """
  Returns a URL (including localhost) to import the given `url` as a notebook.
  """
  def notebook_import_url(url) do
```

Reply:
Can you also please change this code to use your new function? :)

https://github.com/livebook-dev/livebook/blob/main/lib/livebook_app.ex#L87-L90

Reply:
sure @josevalim :)